pkg/ottl/ottlfuncs/README.md

```diff
@@ -316,6 +319,28 @@ Examples:
 
 - `ConvertCase(metric.name, "snake")`
 
+### FNV
+
+`FNV(value)`
+
+The `FNV` Converter converts the `value` to an FNV hash/digest.
+
+The returned type is int64.
+
+The input `value` types:
+* string. If it fails then nil will be returned.
+
+If `value` is another type or parsing failed nil is always returned.
+
+The `value` is either a path expression to a telemetry field to retrieve or a literal.
+
+Examples:
+
+- `FNV(attributes["device.name"])`
+
+
+- `FNV("name")`
```

Comment:
@TylerHelmuth should these functions be upper or lower case? The same for `SHA1` and `SHA256`?

Reply:
I think all uppercase is good